src_api/__init__.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+import os
+
+app = Flask(__name__, template_folder="../templates")
+
+from src_api import views
+
+# logging.basicConfig(level=logging.DEBUG)
+# logging.basicConfig(filename='app.log', level=logging.DEBUG,format=’%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s’)
+
+app.config["APP_FOLDER"] = os.path.abspath(os.path.join(app.root_path, os.pardir))
+app.logger.info("APP FOLDER: " + app.config["APP_FOLDER"])
+
+app.config["UPLOAD_FOLDER"] = app.config["APP_FOLDER"] + "/static/client/wav"

```

Comment:
<img width="602" alt="image" src="https://user-images.githubusercontent.com/93911252/147697562-fb4ac5bb-5d89-4928-9dc0-94428a44df20.png">

an issue with backward and forward slash?

Reply:
I see... the problem is that the path is hard coded and you run the app from windows powershell where the path is handled in a different way than under linux.  
Have you tried to run in git bash? It should not raise any problem...
In the long run, I think we can replace any hard coded paths and use python path libraries like path.join instead. 
It's only a small issue but I cannot troubleshoot as i am running under linux.
In the interest of time do you want to go ahead with the merge and try to update the path statement in a separate dev branch? 
For info the Heroku production environment is linux so the app should deploy correctly.

Reply:
Angelo, I pushed a new version of the dev branch which does not have any hard coded path.. you can pull it and try if it works in powershell..

Reply:
The path-format is now fine, but the file seems unavailable after choosing it (can't see it after choosing it, we can fix it later)